src/script/components/calling/FullscreenVideoCall.tsx

```diff
@@ -249,27 +247,14 @@ const FullscreenVideoCall: React.FC<FullscreenVideoCallProps> = ({
 
   const totalPages = callPages.length;
 
-  const isSpaceOrEnterKey = (event: React.KeyboardEvent<HTMLButtonElement>) =>
-    [KEY.ENTER, KEY.SPACE].includes(event.key);
-
-  const handleToggleCameraKeydown = (event: React.KeyboardEvent<HTMLButtonElement>) => {
-    if (isSpaceOrEnterKey(event)) {
-      toggleCamera(call);
-    }
-
-    return true;
-  };
-
   // To be changed when design chooses a breakpoint, the conditional can be integrated to the ui-kit directly
   const horizontalSmBreakpoint = useMatchMedia('max-width: 680px');
   const horizontalXsBreakpoint = useMatchMedia('max-width: 500px');
   const verticalBreakpoint = useMatchMedia('max-height: 420px');
 
   useEffect(() => {
     const onKeyDown = (event: KeyboardEvent): void => {
-      if (!isEnterKey(event)) {
-        event.preventDefault();
-      }
+      event.preventDefault();
```

Comment:
We `preventDefault` for all key presses, including enter, and add `onKeyDown` events to buttons affected